test/fixtures/bats/empty_name.bats

```diff
@@ -0,0 +1,3 @@
+@test  { true; }
+@test '' { true; }
+@test '	 	 	 ' { true; }
```

Comment:
Perhaps move "tabs and spaces" to individual tests and annotate/comment them for clarity?

Reply:
If I can speak for @mbland here, I think this test is still just ensuring that a whitespace-only test name is still treated as "empty", so I think it still belongs in this set of fixtures.